stdlib/public/Concurrency/Task.swift

```diff
@@ -15,22 +15,39 @@ import Swift
 
 // ==== Task -------------------------------------------------------------------
 
-/// An asynchronous task (just "Task" hereafter) is the analogue of a thread for
-/// asynchronous functions. All asynchronous functions run as part of some task.
+/// A unit of asynchronous work.
 ///
-/// A task can only be interacted with by code running "in" the task,
-/// by invoking the appropriate context sensitive static functions which operate
-/// on the "current" task. Because all such functions are `async` they can only
+/// All asynchronous functions run as part of some task.
+///
+/// Only code that's running as part of the task can interact with that task,
+/// by invoking the appropriate context-sensitive static functions which operate
+/// on the current task. Because all such functions are asynchronous, they can only
 /// be invoked as part of an existing task, and therefore are guaranteed to be
 /// effective.
+/// ◊TR: What do we mean by "effective" above?
 ///
 /// A task's execution can be seen as a series of periods where the task was
-/// running. Each such period ends at a suspension point or -- finally -- the
+/// running. Each such period ends at a suspension point or the
 /// completion of the task.
 ///
 /// These partial periods towards the task's completion are `PartialAsyncTask`.
-/// Partial tasks are generally not interacted with by end-users directly,
-/// unless implementing a scheduler.
+/// Unless you're implementing a scheduler,
+/// you don't generally interacted with partial tasks directly.
```

Comment:
```suggestion
/// you don't generally interact with jobs directly.
```

I believe John is removing the notion of partial tasks and they'll be named "Jobs": https://github.com/apple/swift/pull/36878/files#diff-92484f8fd5d76f2bd84b8509dc599fde13def3c55f342c8a64ffbba2cc9d7c97L7